test/functional/fixtures/api/es-next/navigate-to-and-test-page/test.js

```diff
@@ -9,14 +9,46 @@ describe('[API] t.navigateTo', function () {
     it('Should validate the url argument', function () {
         return runTests('./testcafe-fixtures/navigate-to-test.js', 'Incorrect protocol', { shouldFail: 'true' })
             .catch(function (errs) {
-                expect(errs[0]).contains('The "url" argument specifies a URL that uses an unsupported ftp:// protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.');
-                expect(errs[0]).contains('> 10 |    await t.navigateTo(\'ftp://localhost:3000/fixtures/api/es-next/navigate-to-and-test-page/pages/index.html\');');
+                expect(errs[0]).contains('The specified "ftp://localhost:3000/fixtures/api/es-next/navigate-to-and-test-page/pages/index.html" test page URL uses an unsupported ftp:// protocol. Only relative URLs or absolute URLs with http://, https:// and file:// protocols are supported.');
+                expect(errs[0]).contains('> 48 |    await t.navigateTo(\'ftp://localhost:3000/fixtures/api/es-next/navigate-to-and-test-page/pages/index.html\');');
             });
     });
+
+    it('Should navigate to a relative file url', function () {
```

Comment:
The basic test 'Navigate to another page' is an example of relative HTTP url. BTW I missed tests for scheme-less urls.